mycroft/messagebus/message.py

```diff
@@ -50,6 +101,20 @@ def reply(self, type, data, context={}):
         return Message(type, data, context=new_context)
 
     def publish(self, type, data, context={}):
+        """
+
+        Copy the original context and add passed in context.  Delete
```

Comment:
I'm not actually clear on the goal of this method; I guess it's used to create a new message while preserving the context of the existing message?